bpmn-model/src/main/java/io/zeebe/model/bpmn/validation/zeebe/TimerEventDefinitionValidator.java

```diff
@@ -64,8 +65,8 @@ public void validate(
   private void validateTimeDate(
       TimeDate timeDate, ValidationResultCollector validationResultCollector) {
     try {
-      javax.xml.bind.DatatypeConverter.parseDateTime(timeDate.getTextContent());
-    } catch (IllegalArgumentException e) {
+      ZonedDateTime.parse(timeDate.getTextContent());
```

Comment:
If we move the parsing logic to `TimeDate` then we can reuse this here to ensure it's a valid time data.